encrypt-decrypt.go

```diff
@@ -31,11 +31,21 @@ func Base64MD5Hash(byteStream []byte) string {
 	return base64MD5Hash
 }
 
-// Encrypt bytes with KMS key referenced by resourceName in the format:
-// projects/<projectname>/locations/<location>/keyRings/<project>/cryptoKeys/<key-ring>/cryptoKeyVersions/1
-func EncryptBytes(ctx context.Context, resourceName string, bytesToEncrypt []byte) ([]byte, error) {
-	// Construct the full key URI for Google Cloud KMS
-	//projects/<projectname>/locations/<location>/keyRings/<project>/cryptoKeys/<key-ring>/cryptoKeyVersions/1
+func EncryptBytesWrapper(ctx context.Context, resourceName string, bytesToEncrypt []byte) ([]byte, error){
```

Comment:
I'd keep the old func name EncryptBytes and rename the old EncryptBytes to something like DoEncryptBytes. I'd even make DoEncryptBytes a unexported func(or private method in Java) if unexported func can be mocked in Go